lib/services/local/events.go

```diff
@@ -3221,6 +3223,46 @@ func (p *workloadIdentityParser) parse(event backend.Event) (types.Resource, err
 	}
 }
 
+func newWorkloadIdentityX509RevocationParser() *workloadIdentityX509RevocationParser {
```

Comment:
Can we move the `workloadIdentityX509Revocation`-specific bits to `events_workload_identity_x509_revocation.go` or something? This file is very unwieldy for no real reason.

Reply:
b88b440064